modin/engines/dask/pandas_on_dask_futures/frame/partition_manager.py

```diff
@@ -0,0 +1,55 @@
+from distributed import get_client
+
+
+from modin.engines.base.frame.partition_manager import BaseFrameManager
+from .axis_partition import (
+    PandasOnDaskFrameColumnPartition,
+    PandasOnDaskFrameRowPartition,
+)
+from .partition import PandasOnDaskFramePartition
+from modin.error_message import ErrorMessage
+
+
+class DaskFrameManager(BaseFrameManager):
+    """This class implements the interface in `BaseFrameManager`."""
+
+    # This object uses RayRemotePartition objects as the underlying store.
+    _partition_class = PandasOnDaskFramePartition
+    _column_partitions_class = PandasOnDaskFrameColumnPartition
+    _row_partition_class = PandasOnDaskFrameRowPartition
+
+    @classmethod
+    def get_indices(cls, axis, partitions, index_func=None):
```

Comment:
Since `index_func` has to be a callable,
```suggestion
    def get_indices(cls, axis, partitions, index_func):
```